src/main/java/core/basesyntax/ColorSupply.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupply {
+    private final Random random = new Random();
+
+    public String getRandomColor() {
+        return String.valueOf(Color.values()[random.nextInt(Color.values().length)]).toLowerCase();

```

Comment:
**Use name() for getting String representation of enum constants**
https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#use-name-for-getting-string-representation-of-enum-constants

And split into a few lines to make it easier to read.